README.md

```diff
@@ -1,14 +1,20 @@
 # Corpus Analysis
 
-This is a project to begin compiling a toy corpus of code, focusing on javascript to start.
+This is a project to begin compiling a toy corpus of code, focusing on
+javascript to start.
```

Comment:
we might want to exclude the README from the prettier formatting...I know when writing stuff manually in vim, my lines tended to be very short and it annoyed a lot of people :trollface:

Reply:
i like short lines too, and we're the authors?

or we can change this line in `.prettierrc.json`: `  "proseWrap": "always",`

Reply:
I'd prefer longer lines in the documents, but I'm also fine leaving as is...not a biggie

Reply:
if you have a preference, then it's already stronger than mine.  I'll send this config change as a PR later.